src/main/java/com/gamingmesh/jobs/Placeholders/Placeholder.java

```diff
@@ -429,6 +431,11 @@ public String getValue(UUID uuid, JobsPlaceHolders placeHolder, String value) {
 		    NumberFormat format = NumberFormat.getInstance(Locale.ENGLISH);
 		    return j == null ? "" : format.format(j.getExperience());
 		case user_jmaxexp_$1:
+		    NumberFormat format = NumberFormat.getInstance(Locale.ENGLISH);

```

Comment:
Remove "NumberFormat" before `format`, the current changes showing duplicate variables.

Reply:
moved it outside the cases